core/drivers/atmel_shdwc.c

```diff
@@ -34,6 +34,19 @@
 #define SLOW_CLK_FREQ		32768ULL
 #define DBC_PERIOD_US(x)	DIV_ROUND_UP((1000000ULL * (x)), SLOW_CLK_FREQ)
 
+struct ddrc_reg_config {
+	uint32_t type_offset;
+	uint32_t type_mask;
+};
+
+/*
+ * @shdwc_as    Peripheral SHDWC Always Secure ()
+ */
+struct shdwc_compat {
+	uint8_t shdwc_as;
```

Comment:
What is the meaning of this field? Why `uint8_t` and not `bool`? Please add a comment.